app/views/rent_mailer/new_rent_created.html.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
I think you can only use the body here. There is a html template in the project

Reply:
Also, we usually use slim files (https://github.com/slim-template/slim). This training is not oriented to do front, so i don't mind using erb

Reply:
Ok!.